### PR TITLE
Start file for Olathe with several recommendations

### PR DESCRIPTION
**Pull request by brianamarie**
This pull request adds a file with things to do in Olathe KS. 

@beardofedu What do you think?

### CODE REVIEW

File: briana-olathe-ks.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Olathe, KS 
+
+## Restaurants 
+- Joe's KC BBQ 
+- Kin Lin
+
```
Comment (beardofedu) [inline briana-olathe-ks.md]: Any restaurants with a good selection of 🍺 

I'm also a big fan of 🌯 s, any places to go?

File: briana-olathe-ks.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Olathe, KS
```
Comment (brianamarie) [inline briana-olathe-ks.md]: Oh, Olathe! I know that place!

File: briana-olathe-ks.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Olathe, KS 
+
+## Restaurants 
+- Joe's KC BBQ 
+- Kin Lin
+
+## Things to do 
+- Shawnee Mission Park
+- Kauffman Center for the Performing Arts
+- Sporting KC :soccer:
```
Comment (beardofedu) [inline briana-olathe-ks.md]: I like to do walking tours 🚶 any recommendations?

File: briana-olathe-ks.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Olathe, KS 
+
+## Restaurants 
+- Joe's KC BBQ 
+- Kin Lin
```
Comment (brianamarie) [inline briana-olathe-ks.md]: What should I eat at these restaurants?

**Pull request merged**